<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainGUI</class>
 <widget class="QMainWindow" name="mainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control 2012 - Die Gui für Blöde</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loadPresentation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Presentation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="abortPresentation">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Presentation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextSlide">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>99</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Slide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousSlide">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>99</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous Slide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startPresentation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Presentation</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>loadPresentation</tabstop>
  <tabstop>abortPresentation</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>abortPresentation</sender>
   <signal>clicked()</signal>
   <receiver>mainGUI</receiver>
   <slot>abortTransmission()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPresentation</sender>
   <signal>clicked()</signal>
   <receiver>mainGUI</receiver>
   <slot>loadPresentation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previousSlide</sender>
   <signal>clicked()</signal>
   <receiver>mainGUI</receiver>
   <slot>previousSlide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextSlide</sender>
   <signal>clicked()</signal>
   <receiver>mainGUI</receiver>
   <slot>nextSlide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startPresentation</sender>
   <signal>clicked()</signal>
   <receiver>mainGUI</receiver>
   <slot>startPresentation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>abortPresentation</sender>
   <signal>clicked()</signal>
   <receiver>mainGUI</receiver>
   <slot>stopPresentation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startPresentation()</slot>
  <slot>uploadProgram()</slot>
  <slot>abortTransmission()</slot>
  <slot>nextSlide()</slot>
  <slot>previousSlide()</slot>
  <slot>stopPresentation()</slot>
  <slot>loadPresentation()</slot>
 </slots>
</ui>
